seeds/_sector/dex/_schema.yml

```
@@ -3001,3 +3001,18 @@ seeds:
         token_bought_amount_raw: uint256
         token_sold_amount_raw: uint256
         block_date: timestamp
+
+  - name: solidly_v3_base_base_trades_seed
```

We may rename the seed name as `solidly_base_base_trades_seed`
```suggestion
  - name: solidly_base_base_trades_seed
```